.gitignore

```diff
@@ -4,7 +4,7 @@ coverage/
 docs/_build/
 
 # The client uses Yarn rather than npm to manage the lockfile.
-package-lock.json
+./package-lock.json
```

Comment:
We are using package-lock.json in frontend-shared for the time being. We may add yarn later. This ensures it does not ignore that file.